tests/model_serving/model_server/keda/test_isvc_keda_scaling.py

```
@@ -0,0 +1,121 @@
+            expected_query="vllm:num_requests_running",
+            expected_threshold=2,
+        )
+
+
+@pytest.mark.parametrize(
+    "unprivileged_model_namespace, ovms_kserve_serving_runtime, ovms_raw_inference_service",
+    [
+        pytest.param(
+            {"name": "kserve-raw-route-reconciliation"},
+            RunTimeConfigs.ONNX_OPSET13_RUNTIME_CONFIG,
+            {"name": ModelFormat.ONNX, "model-version": ModelVersion.OPSET13, "model-dir": "test-dir"},
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)